### PR TITLE
Changing the location of several directories

### PR DESCRIPTION
**Pull request by Qudix**
Moving several things from the install directory to the dlls folder

Edit by AL:
Moved to /dlls
```
    imageformats
    platforms
    pythoncore
    QtQuick.2
    styles
    boost_python38-vc142-mt-x64-1_74.dll
    libffi-7.dll
    python38.dll
    pythoncore.zip
```

Moved to /resources
```
    translations
    tutorials
    QtWebEngineProcess.exe
```

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -223,17 +223,19 @@ target_include_directories(${CMAKE_PROJECT_NAME} PRIVATE
...
+		
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/tutorials
+		DESTINATION bin/dlls)
 
 deploy_qt(BINARIES ModOrganizer.exe uibase.dll plugins/bsa_packer.dll)
 
-install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/../qdds.dll DESTINATION bin/dlls/imageformats)
-install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/resources/markdown.html DESTINATION bin/resources)
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/../qdds.dll DESTINATION bin/dlls/QtPlugins/imageformats)
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/resources/markdown.html DESTINATION bin/dlls/resources)
```
Comment (isanae) [inline src/CMakeLists.txt:241]: `LootDialog::createUI()` still refers to `resources/markdown.html` from the root.

**Comment by Holt59** [comment]
The translations are not detected in MO2.

File: src/main.cpp [hunk 2]
```diff
@@ -710,7 +708,8 @@ int main(int argc, char *argv[])
 
   TimeThis tt("main() to doOneRun()");
 
-  SetThisThreadName("main");
+  env::prependToPath(QDir::toNativeSeparators(
```
Comment (isanae) [inline src/main.cpp:711]: Looks like a bad merge, `SetThisThreadName()` is missing. It's also redundant, it's already done in `addDllsToPath()`.

**Comment by isanae** [comment]
> preloadDll is necessary to ensure Qt doesn't whine about the OpenSSL libraries not being in both the install directory and the dlls folder

Note that `preloadDll()` was removed in my PR because it wasn't necessary at the time. This might have broken some of your stuff.

**Comment by Qudix** [comment]
Working on it

**Comment by Al12rs** [comment]
This requires another rebase due to the last PR from isa. Make sure to use the appconfig for the translations path, like what isa did in his pr.

**Review by Al12rs — APPROVED** [review]

**Pull request merged**